### PR TITLE
Milestone2: Portfolio mobile version

### PR DESCRIPTION
**Pull request by shahira-sadat**
Work Section and About Myself Section Added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,7 +43,154 @@ <h1>I'm Shahira <br> Glad to see <br>you!</h1>
           </ul>
         </section>
 
-    </main>
+        <div class="card-works">
```
Comment (Chimwemwe-127001) [inline index.html:46]: - [ ] I suggest you use a `<section>` tag for this container. The `<section>` element is a semantic element; it provides meaning to both user agents and humans about the enclosed content — specifically, a section of the document. The sub-containers in this container can be just `divs`.

Comment (Chimwemwe-127001) [reply index.html:46]: - [ ] As the requirements suggest, In order to lay out the elements in the `"Works"` section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

File: index.html [hunk 2]
```diff
@@ -43,7 +43,154 @@ <h1>I'm Shahira <br> Glad to see <br>you!</h1>
...
+                    </li>
+                    <li><i class="fa fa-chevron-right" aria-hidden="true"></i></li>
+                  </ul>
+                  <ul class="skills">
+                    <li class="flex-items">
+                      <p>Skills</p>
+                    </li>
+                    <li><i class="fa fa-chevron-right" aria-hidden="true"></i></li>
+                  </ul>
+                </div>
```
Comment (Chimwemwe-127001) [inline index.html:191]: - [ ] For this part of the portfolio, I suggest you closely follow the figma design.

Your design:
image

Figma design:
image

**Review by Chimwemwe-127001 — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
### Hi @shahira-sadathi
Good job so far! :tada: 

There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand: 

so close

## Highlights :+1: 
- Repo looks professional :briefcase:
- README.md looks professional :briefcase:

## Required Changes ♻️
***Check the comments under the review.***

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi Team👋 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**